CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+## [0.8.0] - 2024-10-03
+
+### Breaking
```

Comment:
could you also add the instructions to docs
```
 wh1.update({eventTypeFilter: {addresses:["0x40A28c0fCc0BE09400bB89CdF556Cd8C4eF1c165","0x6f03b3Df22F0C57A4477EEAc3a49c2Bc4EAe2206"]}})

```

Reply:
do you mean this one ? https://github.cbhq.net/solutions-architecture/docs/pull/715/files#diff-e948e3fa52983fe8dee3cacd990a72989a39f539973f07fca0f2065dc7a0577dR15

Reply:
if that is the place to put documentation then this change is documented. cc: @jazz-cb